<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>39</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Showcard Gothic</family>
        <pointsize>22</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>MENU</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejercicio1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>220</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejercicio 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>310</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejercicio 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>400</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejercicio 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>490</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Finalizar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese un titulo a su tarea:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputTitulo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="inputDescripcion">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>351</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="inputFecha">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="date">
       <date>
        <year>2024</year>
        <month>8</month>
        <day>9</day>
       </date>
      </property>
     </widget>
     <widget class="QComboBox" name="tipoTarea">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>130</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Diaria</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semanal</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnAgregarTarea">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese un descripcion a su tarea:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listaTareas">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>300</y>
        <width>601</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Descripcion de la tarea:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>500</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>510</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="figuraTipo">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Círculo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rectángulo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triángulo</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Selecciona el tipo de figura:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAgregarFigura">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>100</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Figura</string>
      </property>
     </widget>
     <widget class="QWidget" name="canvasWidget" native="true">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>190</y>
        <width>481</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid red;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modificador de Texto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>510</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="TE_Editor">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>90</y>
        <width>481</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="negritaButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NEGRITA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cursivaButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CURSIVA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="normalButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NORMAL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="subrayadoButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>380</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SUBRAYADO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese Talla de Producto:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese Cantidad de Producto:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>90</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-style:italic; color:#ffff00;&quot;&gt;-&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt; Para hacer una modificacion, seleccione la fila que desea editar y en las mismas casillas edite los valores, aunque estos no sean del mismo. Al terminar de editar nuevamente presione modificar otra vez y sus valores nuevos se mostraran.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; font-style:italic;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-style:italic; color:#ffff00;&quot;&gt;-&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt; Para eliminar seleccione la casilla deseada, oprima&amp;quot;Eliminar&amp;quot; y listo.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>210</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese Marca de Producto:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateCaducidadEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="date">
       <date>
        <year>2024</year>
        <month>8</month>
        <day>9</day>
       </date>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTipoProducto">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Electronico</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ropa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alimento</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="nombreEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMarca">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>230</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese Precio de Producto:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>-10</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SISTEMA DE GESTION DE INVENTARIO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SBprecioEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100000.990000000005239</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese Nombre de Producto:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 0, 0);</string>
      </property>
      <property name="text">
       <string>- De NO aplicar dicha casilla a producto,</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxCantidad">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTalla">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>230</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese TIPO DE PRODUCTO:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>SOLO SI ES ALIMENTO</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese Fecha de Caducidad de Producto:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>631</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAgg">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>340</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>AGREGAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelete">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>410</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>ELIMINAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMod">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>490</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>MODIFICAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>490</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
